db/schema.rb

```diff
@@ -11,15 +11,30 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160127035545) do
+ActiveRecord::Schema.define(version: 20160127193752) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "games", force: :cascade do |t|
```

Comment:
Let's wait to merge this until the other one is merged. We might be able to get rid of this extra stuff.